packages/kit/src/strategies/index.ts

```diff
@@ -51,7 +51,7 @@ function getStrategyTypeFromName(strategyName: string, chainId: number) {
 // Helper function to find matching contract from name or address
 function reduceSupportedChains(
   chainId: number,
-  compare: (args: [key: string, address: Address]) => boolean,
+  compare: (args: [key: string, address: Address | number]) => boolean,
```

Comment:
address: Address | number?

Reply:
Typescript was complaining, because of this:

@gitcoin/gitcoin-chain-data/dist/types.d.ts

export type TContracts = {
    multiRoundCheckout: Address;
    quadraticFunding: Address;
    directGrants: Address;
    directAllocationPoolId?: number;
    directAllocationStrategyAddress?: Address;
};

the values of an object TContracts can be Address or number.
not sure why the poolId is in that object though, but I didnt work on that repo yet.